src/ecuacion/SistemaDeInventario.java

```diff
@@ -0,0 +1,281 @@
+				System.out.println("Si- 1 \t No-0");
+				opc = scan.nextInt();
+				if (opc == 1) {
+					programa = true;
+				}
+
+			} else {
+				System.out.println("Opcion incorrecta");
+			}
+
+		} while (programa == false);
+
+	}
+	// Metodos
+	static void agregarProducto(int cantProductos) {
+		for(int i =0;i <cantProductos; i++){
+			scan.nextLine();
+			System.out.println("Ingrese el nombre del producto");
+			nombreProducto[cantTotalProductos]= scan.nextLine();
+			System.out.println("Ingrese el codigo del producto");
+			String codigo = scan.nextLine();
+			for(int j=0;j<codigoProducto.length;j++){
+				if(codigo.equals(codigoProducto[j])) {
+					while(codigo.equals(codigoProducto[j])) {
+						System.out.println("Codigo repetido, ingrese un codigo valido");
+						codigo= scan.nextLine();
+					}
+				}
+			}
+			codigoProducto[cantTotalProductos] = codigo;
+			System.out.println("Ingrese el precio del producto");
+			precioProducto[cantTotalProductos] = scan.nextDouble();
+			// Mayor y menor precio
+			if (precioProducto[cantTotalProductos] > productoMasCostoso) {
+				productoMasCostoso = precioProducto[cantTotalProductos];
+				nombreMasCostoso = nombreProducto[i];
+			}
+			if (precioProducto[cantTotalProductos] < productoMenosCostoso) {
+				productoMenosCostoso = precioProducto[cantTotalProductos];
+				nombreMenosCostoso = nombreProducto[i];
+			}
+			System.out.println("Ingrese la cantidad de articulos del producto ingresado");
+			cantidadProducto[cantTotalProductos] = scan.nextInt();
+			// Cantidad mayor y menor de producto
+			if (cantidadProducto[cantTotalProductos] > mayorCantArticulos) {
+				mayorCantArticulos = cantidadProducto[cantTotalProductos];
+				nombreMayorCant = nombreProducto[i];
+			}
+			if (cantidadProducto[cantTotalProductos] < menorCantArticulos) {
+				menorCantArticulos = cantidadProducto[cantTotalProductos];
+				nombreMenorCant = nombreProducto[i];
+			}
+
+			cantTotalProductos++;
+		}
+	}
+
+	static void configuracion() {
+		System.out.println("Configuracion");
+		System.out.println("1- IVA");
+		System.out.println("2- Tasa USD/BS");
+		System.out.println("3-Salir");
+		int opcion = scan.nextInt();
+		if (opcion == 1) {
+			System.out.println("IVA");
+			System.out.println("Tasa actual: " + IVA + "%");
+			System.out.println("Ingrese la nueva tasa:");
+			IVA = scan.nextInt();
+			System.out.println("Nueva tasa: " + IVA + "%");
+
+		} else if (opcion == 2) {
+			System.out.println("TASA USD/BS");
+			System.out.println("Tasa actual: " + tasaUsdBs);
+			System.out.println("Ingrese la nueva tasa");
+			tasaUsdBs = scan.nextInt();
+			System.out.println("Nueva tasa: " + tasaUsdBs);
+		}
+	}
+
+	static void estadisticas() {
+		int cantTotalArticulos = 0;
+		double cantTotalPrecios = 0;
+		for (int i = 0; i < cantTotalProductos; i++) {
+			cantTotalArticulos += cantidadProducto[i];
+			cantTotalPrecios += precioProducto[i];
+		}
+		for (int k = 0; k < cantTotalProductos; k++) {
```

Comment:
por qué K?, si i está disponible para usar ya que pertenece al for de arriba y este for está fuera de scope.